src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -855,7 +855,7 @@ namespace FourPipeBeam {
             Real64 minFlow(0.0);
             Real64 maxFlowCool(0.0);
             minFlow = std::min(state.dataEnvrn->StdRhoAir * originalTermUnitSizeMaxVDot,
-                               state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesOAFlow * state.dataEnvrn->StdRhoAir);
+                               state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).MinOA * state.dataEnvrn->StdRhoAir);
```

Comment:
I don't even see where DesOAFlow is set in current develop. So was this always 0 prior to this change?

Reply:
Now I am confused, I don' even see where DesOAFlow is a member of ZoneSizingData which is where TermUnitFinalZoneSizing comes from.

    EPVector<DataSizing::ZoneSizingData> TermUnitFinalZoneSizing; // Final data for sizing terminal units (indexed per terminal unit)

Reply:
@mjwitte was deleting this from the sizing structs when working on `Space` (mentioned in comment above) so this was getting removed anyways.  Looks like he has done this at this point.